### PR TITLE
Auto Activate Drop-Down Property editor

### PR DESCRIPTION
**Pull request by theanuradha**
https://github.com/debrief/debrief/issues/1694

### CODE REVIEW

**Comment by IanMayo** [comment]
Can you give me some background to this?

**Comment by theanuradha** [comment]
@IanMayo  as of #1694 On property editors with combo (drop down) had to click two times . now it show drop down soon as user activate editor

**Comment by theanuradha** [comment]
https://www.youtube.com/watch?v=nUNtP6pkCSA

File: org.mwc.cmap.core/src/org/mwc/cmap/core/property_support/TagListHelper.java [hunk 1]
```diff
@@ -35,7 +36,21 @@ public TagListHelper(final String[] theTags,
 
 	public CellEditor getCellEditorFor(final Composite parent)
 	{
-		return new ComboBoxCellEditor(parent, _theTags);
+		return new ComboBoxCellEditor(parent, _theTags, SWT.READ_ONLY){
```
Comment (IanMayo) [inline org.mwc.cmap.core/src/org/mwc/cmap/core/property_support/TagListHelper.java]: Hmm, it seems strange that we show the Combo Box in SWT.READ_ONLY mode.  I can see it's working, but could you explain why it's READ_ONLY?

Comment (theanuradha) [inline org.mwc.cmap.core/src/org/mwc/cmap/core/property_support/TagListHelper.java]: I made it readonly mode as if not user can enter any text and it throw error when user do that

Comment (IanMayo) [inline org.mwc.cmap.core/src/org/mwc/cmap/core/property_support/TagListHelper.java]: Aah, to prevent "free text edit".  I see. Thanks :-)

**Pull request merged**